### PR TITLE
add frankie-lib as submodule instead of jar

### PR DESCRIPTION
**Pull request by bepo00**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -22,6 +22,12 @@ android {
     }
 
     signingConfigs {
+        release {
+            storeFile file('../keystore/debug.keystore')
+            storePassword = 'KSr3#V^^heD7g1mIO!m9'
+            keyAlias 'debug'
+            keyPassword = 'Ow0%I!TUwDONAejpS^XF'
+        }
```
Comment (bepo00) [inline app/build.gradle:30]: this one should work for you I used the local one as I need for debug

File: app/proguard-rules.pro [hunk 2]
```diff
@@ -1,21 +1,82 @@
...
+
+# ----------------- reflection -------
+-keep class kotlin.Metadata { *; }
+-keep class kotlin.reflect.** { *; }
+
+# ------------------ KEEP API classes ----------
+-keep class com.frankie.app.api.** { *; }
+
+# -------------------KEEP expression manager
+-keep class com.frankie.expressionmanager.** { *; }
```
Comment (bepo00) [inline app/proguard-rules.pro:82]: Still most important part is to be finished

File: app/src/main/java/com/frankie/app/business/responses/ResponseRepository.kt [hunk 3]
```diff
@@ -4,11 +4,10 @@ import com.frankie.app.business.survey.SessionManager
 import com.frankie.app.db.ResponseDao
 import com.frankie.app.db.model.Response
 import com.frankie.expressionmanager.model.ResponseEvent
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.flow
 
 interface ResponseRepository {
     suspend fun getResponses(surveyId: String): List<Response>
```
Comment (askoura) [inline app/src/main/java/com/frankie/app/business/responses/ResponseRepository.kt:9]: We actually only use this to get the unsynced responses... can you change this method to getunsynced responses?

**Comment by askoura** [comment]
- Let's create 2 signing configs that we share in the git repo... one for release and another for debug
- We want to make sure google services.json is part of the repo... so we remove it from .gitignore

**Pull request merged**